flow/e2e/clickhouse/peer_flow_ch_test.go

```diff
@@ -1068,15 +1068,20 @@ func (s ClickHouseSuite) Test_Unprivileged_Postgres_Columns() {
 		CREATE TABLE IF NOT EXISTS %s (
 			id SERIAL PRIMARY KEY,
 			key TEXT NOT NULL,
-			"se'cret" TEXT
+			"se'cret" TEXT,
```

Comment:
note this is being excluded so doesn't fully test `'` handling

You may want to see the WeirdTable function which was test meant to look into these (but skipped columns since those didn't work)